package.json

```diff
@@ -15,7 +16,7 @@
     "dev": "tsc --watch",
     "lint": "eslint . --ext .js,.ts,.vue && npm run csslint",
     "lintfix": "eslint . --ext .js,.ts,.vue --fix && npm run csslintfix",
-    "setup": "npm ci && npm run clean && lerna clean --yes && lerna bootstrap --hoist",
```

Comment:
Could you explain why `npm run build:backend` is added here?
Also, `lerna bootstrap --hoist` is removed because it's no longer available or no longer needed?

Reply:
HMMMMMMM
I am not 100% sure on why I need npm run build:backend.
1. lerna bootstrap --hoist is deprecated. Npm workspaces will hoist the node_modules now so this is no longer needed.
2. Without npm run build:backend, it does not work (probably due to js files not being generated somehow) but this is completely not related to lerna boostrap --hoist lol.

Reply:
The current setup works because in packages/core, the "prepare" command is defined in package.json. So it seems to be that when lerna bootstrap is activated, the "prepare" command that calls "npm run build" is ran in packages/core, which handles the building of typescript backend.

Quoting ChatGPT's potentially erroneous explanation:
> Yes, by default, when you run lerna bootstrap, it will run the prepare script in each package's package.json file, if it is defined. The prepare script is typically used to perform any necessary build steps or other preparation tasks for a package before it is published or used by other packages.

> When Lerna runs npm install for each package, it will also run the prepare script if it is defined. This means that any necessary build steps or other preparation tasks will be executed during the installation process, ensuring that the package is ready for use or publication.

Can we find out how to do so without lerna, or if there's a new command that does that? There's no need to call `npm run build:backend` here when we already specify it in the `prepare` step.

Reply:
bootstrap is deprecated.
From what I read, npm i calls prepare in workplace but we do not want to do that as npm i may break MarkBind.

So, I guess we can run prepare manually instead of npm run build:backend.

Reply:
when you say "manual" do you mean it or are you referring to invoking the command as part of "setup"?

Reply:
Yes to adding that to part of setup.